aiida/cmdline/commands/cmd_import.py

```diff
@@ -224,8 +224,9 @@ def _import_archive(archive: str, web_based: bool, import_kwargs: dict, try_migr
                 echo.echo_info(f'incompatible version detected for {archive}, trying migration')
                 try:
                     migrator = get_migrator(detect_archive_type(archive_path))(archive_path)
-                    archive_path = temp_folder.get_abs_path('migrated_archive.zip')
-                    migrator.migrate(EXPORT_VERSION, archive_path, out_compression='zip')
+                    archive_path = migrator.migrate(
+                        EXPORT_VERSION, None, out_compression='none', work_dir=temp_folder.abspath
```

Comment:
I thought we were always going to specify the function arguments ;-)

Reply:
"do as I say, not as I do" 😆
but anyway here the `*` in the `migrate` method arguments is placed after the version and filename, which are variables that will always be required for a migration.
I don't think it's necessary for absolutely all arguments but certainly for most (if not all) optional keyword arguments and/or when there is more than 1 or 2 arguments